src/hub.html

```diff
@@ -105,7 +105,7 @@
             </template>
 
             <template id="remote-avatar-template">
-                <a-entity ik-root player-info>
+                <a-entity ik-root player-info hand-poses="mixer: .RootScene">
```

Comment:
I think `hand-poses` should jsut be on RootScene so it doesnt have to try and deal with any async stuff to get the mixer